scaling_performance/optimizing_storage.adoc

```diff
@@ -175,13 +175,64 @@ to nodes to support a healthy cluster.
 == Choosing a Docker Graph Driver
 
 Docker stores images and containers in a graph driver (a pluggable storage
-technology), such as `DeviceMapper`, `Overlay`, and `Btrfs`. Each have advantages and
-disadvantages. For example, `Overlay` is faster than `DeviceMapper` at starting and
-stopping containers, but is not Portable Operating System Interface for Unix (POSIX) compliant because of the architectural
-limitations of a union file system, and does not yet support SELinux.
+technology), such as `DeviceMapper`, `Overlay`, and `Btrfs`. Each have
+advantages and disadvantages. For example, `Overlay` is faster than
+`DeviceMapper` at starting and stopping containers, but is not Portable
+Operating System Interface for Unix (POSIX) compliant because of the
+architectural limitations of a union file system, and does not yet support
+SELinux.
+
+For more information about `Overlay`, including supportability and usage
+caveats, see the
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/7.3_Release_Notes/index.html#technology_previews_file_systems[Red
+Hat Enterprise Linux (RHEL) 7 Release Notes].
+
+.Docker Graph Driver Comparisons
+|===
+|Name |Description |Benefits |Limitations |Defaults
+
+|Device Mapper loop-lvm
+|Uses the device mapper thin provisioning module (dm-thin-pool) to implement copy
+on write (CoW) snapshots. For each device mapper graph location, thin pool is
+created based on two block devices, one for data and one for metadata. By
+default, these block devices are created automatically by using loopback mounts
+of automatically created sparse files.
+|Benefits go here
+|Limitations go here
+|100GB pool (total max across all containers)
+
+|Device Mapper direct-lvm
+|Also uses LVM, device mapper, and the dm-thinp kernel module. It differs by removing
+the loopback device, talking straight to a raw partition (no filesystem).
+|There are measurable performance advantages at moderate load and high density.
+|Limitations go here
+|Defaults got here
+
+|btrfs
+|A CoW filesystem for Linux aimed at implementing advanced features while also
+focusing on fault tolerance, repair, and easy administration.
+a|* Meets the basic requirements of supporting CoW.
+* Moderately performant.
+* Actively developed.
+a|* Does not currently support SELinux.
+* Does not allow page cache sharing.
+|Defaults go here
+
+|OverlayFS
+|Combines a lower (parent) and upper (child) filesystem and a
+workdir (on the same filesystem as the child). The lower filesystem is the base
+image, and when you create new Docker containers, a new upper filesystem is
+created containing the deltas.
+a|* Fast (faster than Device Mapper at starting and stopping containers)
```

Comment:
I would state here the startup time difference between devicemapper and overlay is generally less than 1 second.